cli/flox-activations/src/activations.rs

```diff
@@ -106,7 +112,7 @@ pub struct Activation {
     /// may change.
     /// We generate a UUID so that we have something convenient to pass around
     /// and use as a directory name.
-    id: Uuid,
+    id: String,
```

Comment:
suggestion nb: I'd appreciate a bit more type safety, string is a pretty broad type and we do have invariants as to how these ids are created.
A `[u8; 8]` based solution would also be `Copy`able, allowing us to avoid explicit cloning.

Reply:
If it's convenient to use a `[u8; 8]` wherever we're using an ID that would be fine by me, but it feels like it's trending a bit towards type safety overkill